zeebe/clients/java/src/main/java/io/camunda/zeebe/client/api/command/UpdateUserTaskCommandStep1.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.command;
+
+import io.camunda.zeebe.client.api.response.UpdateUserTaskResponse;
+import java.util.List;
+
+public interface UpdateUserTaskCommandStep1 extends FinalCommandStep<UpdateUserTaskResponse> {
+
+  /**
+   * Set the custom action to update the user task with.
+   *
+   * @param action the action value
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 action(String action);
+
+  /**
+   * Set the due date to set in the user task.
+   *
+   * @param dueDate the due date to set
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 dueDate(String dueDate);
```

Comment:
@koevskinikola and @ana-vinogradova-camunda, any thoughts on the validation of the provided data?
Should we
* Validate the format here?
* Validate the format in the REST controller in the gateway?
* Never validate the data format?

We don't interpret this data anywhere in the engine. We do however rely on the right format in downstream applications like Tasklist. 

We could manifest this in the REST API with OpenAPI, using a [format](https://swagger.io/docs/specification/data-models/data-types/#format) like `date-time`. Validating it manually might however yield more flexibility. We should still document it in the endpoint then.

Reply:
I extracted this into a separate issue, no need to block progress here: https://github.com/camunda/zeebe/issues/16734